source/source-connector/fundamentals/specify-schema.txt

```diff
@@ -182,6 +182,23 @@ following options:
    output.format.value=schema
    output.schema.infer.value=true
 
+The source connector can infer schemas for incoming documents that
+contain nested documents stored in arrays. Starting in Version 1.9 of the
+{+connector+}, schema inference does not default to ``string`` type
+assignment if there are differences between nested documents
+described by the following cases:
+
+- A field is present in one document but missing in another.
+- A field is present in one document but ``null`` in another.
+- A field is an array with elements of any type in one document but
+  has additional elements or elements of other data types in another.
+- A field is an array with elements of any type in one document but an
+  empty array in another.
+
+If field types conflict between nested documents, the connector
+pushes the conflict down to the schema for the field and defaults to
+``string`` type assignment.
```

Comment:
[suggestion] - Minor nitpick here but I feel like there should be an article before ``string``.

Suggestion: 
If field types conflict between nested documents, the connector pushes the conflict down to the schema for the field and defaults to a ``string`` type assignment.